azure-iot-device/azure/iot/device/provisioning/sk_registration_client.py

```diff
@@ -0,0 +1,80 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module contains one of the implementations of the Registration Client which uses Symmetric Key authentication.
+"""
+import logging
+from threading import Event
+from .registration_client import RegistrationClient
+from .models.registration_result import RegistrationResult
+from .internal.polling_machine import PollingMachine
+
+logger = logging.getLogger(__name__)
+
+
+class SymmetricKeyRegistrationClient(RegistrationClient):
```

Comment:
>SymmetricKeyRegistrationClient [](start = 6, length = 30)

since the parent class is to be renamed I have renamed this one as well to `SymmetricKeyProvisioningDeviceClient`.